tools/wptserve/wptserve/utils.py

```diff
@@ -72,6 +72,7 @@ def is_bad_port(port: int) -> bool:
     Bad port as per https://fetch.spec.whatwg.org/#port-blocking
     """
     return port in [
+        # 0, Blocking this here breaks WPT infrastructure.
```

Comment:
Pretty confident that this _doesn't_ cause any of the breakage, so:

```suggestion
        0,
```